examples/config.yml

```
@@ -30,6 +30,14 @@ modules:
         active: 1         # static value
         count: '{.count}' # dynamic value
         boolean: '{.some_boolean}'
+    - name: example_count
+      type: countbylabel
+      help: Example of count json labels
+      path: '{.values[*].state}'
+      labels:
+        environment: beta # static label
+        state: '{}'       # dynamic label
+      mincount: 1
```

This mincount is not exactly obvious without reading the code. Let's document it in the example config. Also, for the config, maybe a nicer name.

```suggestion
      # The minimum count of labels required to expose the metric. Defaults to 1.
      minimumCount: 1
```

---

@SuperQ @rustycl0ck please, take a look at this issue again